aws/components/apigateway/setup.ftl

```diff
@@ -473,6 +477,18 @@
         [/#if]
     [/#if]
 
+    [#-- hamlet doesn't create the execution log group, but can manage any subscriptions   --]
+    [#-- Note that subscriptions will only be created on the second and subsequent stack   --]
+    [#-- execution, i.e. when the epilogue script has been successfully run at least once. --]
+    [#if getExistingReference(executionLgId)?has_content]
```

Comment:
Might be good to add an @warn that you need to run a second pass

Reply:
Yup - good idea.